requests/CfxUsdPrice.js

```diff
@@ -0,0 +1,77 @@
+import * as Witnet from "witnet-requests"
```

Comment:
So cool that you included the CFX price feed! :heart_eyes:

Reply:
I just saw your PR, so we would be merging two for the price of one :-)